OpenRA.Game/Settings.cs

```diff
@@ -251,6 +251,14 @@ public class PlayerSettings
 		public Color[] CustomColors = Array.Empty<Color>();
 	}
 
+	public class GameplaySettings
+	{
+		[Desc("Sets the Auto-save frequency, in seconds")]
+		public int AutoSaveInterval = 120;
+		[Desc("Sets the AutoSave number of max files to bes saved on the file-system")]
+		public int AutoSaveFileNumber = 10;
```

Comment:
actually could these options be moved into the `GameSettings` right bellow? (and the class deleted)

This also fails to save into the settings file as it's not mentioned in the settings constructor, moving is one way to fix it

Reply:
I've added `Gameplay` to the dictionary and now it saves the settings.

Reply:
`SinglePlayerAutoSaveInterval` or `SinglePlayerGameSettings`?

I first thought this was for the map editor until i read the issue.

Reply:
> `SinglePlayerAutoSaveInterval` or `SinglePlayerGameSettings`?
> 
> I first thought this was for the map editor until i read the issue.

Uh, sure, why not? I guess if the class' name is SinglePlayerGameSettings, then maybe `AutoSaveInterval` is clear enough in that context? Because I find it a bit redundant to call the class `SinglePlayerGameSettings` and then also the `AutoSaveInterval` – `SinglePlayerAutoSaveInterval`

Reply:
Yeah i meant one of the two. But better if others judge/decide. SinglePlayerGameSettings seems cleaner to me.